Terraform/userdata.sh

```diff
@@ -172,6 +172,18 @@ server {
 }
 EOF
 
+mkdir -p /var/opt/jfrog/artifactory/etc/info
+cat <<EOF >/var/opt/jfrog/artifactory/etc/info/installer-info.json
+{
+  "productId": "JFrog_TerraformInstaller_Mysql/1.0.0"
+  "features": [
+  {
+    "featureId": "Artifactory/${artifactory_version}",
+    "attributes": {}
```

Comment:
You can skip the optional empty attribute

Reply:
line removed